debian/changelog

```diff
@@ -1,3 +1,9 @@
+kano-toolset (4.2.1-0) unstable; urgency=low
+
+  * Fixed wireless connection on boot for Kano 4.2.0 release
```

Comment:
I'd rewrite this as `Switched to using the original wpa supplicant configuration file`. The version mentioned here is actually 4.2.1 like in the line above.